### ISSUE TITLE
Custom ‘fullscreen_back_button’ button events.

### ISSUE THREAD

**Issue by babysource** [googlevr/gvr-android-sdk#205]
I would like to suggest that hope to Custom ‘fullscreen_back_button’ button events.

**Comment by nathanmartz** [comment]
I'm sorry, but I don't really understand the request. Can you rephrase it?

**Comment by babysource** [comment]
In some scenarios, We need to do something before or after at the time of back. So, I hope to be able to customize the  ‘fullscreen_back_button’  button events.

**Comment by nathanmartz** [comment]
In the latest SDK, you can procedurally change between fullscreen and VR and embedded. If you want to add your own buttons then you can do whatever you like and then programmatically change the viewing mode.

**Comment by electmk** [comment]
@nathanmartz 
@sigmaxipi 
I use panoWidgetView.setDisplayMode(3) to Direct entry Vr mode but still appears (fullscreen_back_button) which opens (pano fullscreen) .. I would like to handle (fullscreen_back_button) button event to finish and back to previous activity
